tests/methods/fci-7/input.dat

```diff
@@ -17,14 +17,16 @@ set {
   scf_type pk
   reference rohf
   e_convergence 12
+  docc [3,0,0,0]
```

Comment:
Why change the test case?

Reply:
This test case is potentially ill-defined. The SCF solution was actually converging on a B1 state and the FCI was instead computing a B2 solution. This is because for this test case the B1 and B2 MOs are degenerate. The new code in the HF class picked up the discrepancy in the symmetry of the two solutions. Now, since there is only one set of p orbitals in the basis, this mixed B1/B2 computation gives the same as doing both a HF and FCI computation on the same state (say B1), however, that's just a coincidence. These extra options now precisely define the test case (without changing the result).